_posts/2021-04-02-shared-db-deprecation-notice.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Shared Database Deprecation Notice
+date: April 2, 2021
+---
+
+_This is a repost of our StatusPage notice, originally posted August 7th, 2020_
+
+# Important updates and enhancements to cloud.gov’s relational database service
+
+This message is to let you know about recent and upcoming changes to cloud.gov’s relational database services, which include the following:
+
+- Deprecation of the shared instance plans
+- Scaling down of under-utilized instances
+- New service plan offerings
+- Ability to modify existing service instances
+- Documentation updates on our website
+
+
+## What is happening and what is the impact to cloud.gov customers?
+
+As of January 24, 2021, shared instance plans will no longer be available as a choice for creating new database instances. Customers will only be able to choose from the dedicated service plans. Sandbox accounts will only have access to the micro-psql and small-mysql service plans.
+
+Within 14 days of this notice, we will also begin scaling down under-utilized database instances to smaller service plans and asking if unused instances are still needed. We will initially target databases with 10 or fewer recent concurrent connections and low CPU usage. Applications that are dependent on database availability may experience a short period of unavailability during the scale operation; please reach out to our team with your org, space, and service name if there are days we should avoid resizing deployed instances. As a cost-recoverable program, we are tasked with being responsible stewards for our operating costs and this will allow us to keep costs under control.
+
+
+## What do cloud.gov customers need to do?
+
+At this time, customers who are currently running shared database server instances will need to begin planning a migration to a dedicated instance plan. To perform a migration, a customer must export their existing database and restore it into a new instance created with a dedicated service plan. We have instructions on how to export a database on our website and will send out more information on how to perform the restore in the near future.
```

Comment:
Should we add a link in this paragraph to https://cloud.gov/docs/services/relational-database/#access-the-data-in-the-database?

Reply:
My first thought was "heck yes," but the goal here was to capture the statuspage post. My assumption (which could be wrong) was that it would be best to capture it unedited, so I'm not 100% sure